src/app.ts

```diff
@@ -74,7 +74,7 @@ if (process.env.SENTRY_DSN) {
 await configure({
     sinks: { console: getConsoleSink() },
     filters: {},
-    loggers: [{ category: 'fedify', sinks: ['console'], level: 'debug' }],
+    loggers: [{ category: 'fedify', sinks: ['console'], level: null }],
```

Comment:
```suggestion
    loggers: [{ category: 'fedify', sinks: ['console'], level: 'error' }],
```

maybe?

Reply:
`'warning'` might be better than `'error'` as Fedify logs deprecation messages as warnings.